src/rt-app.c

```diff
@@ -712,6 +748,11 @@ static void create_cpuset_str(cpuset_data_t *cpu_data)
 	}
 
 	cpu_data->cpuset_str = malloc(size_needed);
+	if (!cpu_data->cpuset_str) {
```

Comment:
This unrelated as well?

Reply:
While fixing unique timer, missing error path has been highlighted. All this are in the 1st patch